packages/frontEnd/src/views/categoryDetails/filters/FilterByCities/FilterByCitiesForm.tsx

```diff
@@ -18,6 +37,8 @@ export const FilterByCitiesForm = () => {
               id="los-angeles"
               className="form-checkbox rounded"
               type="checkbox"
+              onChange={(event) => onChange(event, "Los Angeles")}
```

Comment:
It seems to me that the comparison must be dynamic, because the city will change with the number of ads